tests/packer_snapshot_spec.lua

```diff
@@ -0,0 +1,34 @@
+local packer = require("packer")
+local path = require("plenary.path")
+local config = {
+    snapshot_path = vim.fn.stdpath("cache").."/packer/"
+}
+
+describe("Packer snapshot tests", function()
+  after_each(function()
+    packer.reset()
+  end)
+
+  it("PackerSnapshot: ", function ()
+    local spec = {"tjdevries/colorbuddy.vim"}
+    local short_name = 'colorbuddy'
+    local snapshot_name = 'test'
+    local test_path = path.joinpath(config.snapshot_path, snapshot_name)
+    local expected = {}
+    expected.colorbuddy = '87c80e3'
+    packer.startup(spec)
+    packer.install()
+    packer.snapshot(snapshot_name)
+    describe("snapshot exists", function ()
```

Comment:
Describe blocks shouldn't be nested within `it` blocks, the describe block should be a top level grouping of what functionality you are testing. Within which each test case should be in an it block.

See [here](http://olivinelabs.com/busted/#overview) which is essentially the library that plenary uses/copied